packages/turf-voronoi/index.js

```diff
@@ -0,0 +1,45 @@
+import lineStringToPolygon from '@turf/linestring-to-polygon';
+import { lineString } from '@turf/helpers';
+import d3voronoi from 'd3-voronoi';
+
+function und3ify(polygon) {
+    return lineStringToPolygon(lineString(polygon.slice(0)));
+}
+
+/**
+ * Takes a FeatureCollection of points, and a bounding box, and returns a FeatureCollection
+ * of Voronoi polygons. 
+ * 
+ * The Voronoi algorithim used comes from the d3-voronoi package.
+ *
+ * @name voronoi
```

Comment:
@stevage this should be `turfVoronoi`

Reply:
Are you sure about this? I've made this change, but Travis is failing and saying "not ok 12 voronoi is missing from index.js". Every other module I've looked at uses a short name that matches the function name.

Reply:
@stevage I believe it's referring to the main `@turf/turf` [`index.js`](https://github.com/Turfjs/turf/blob/master/packages/turf/index.js), might be wrong though

Reply:
`@name voronoi` is good.

`turfVoronoi` is incorrect.